packages/desktop-client/src/components/reports/ReportOptions.ts

```diff
@@ -363,7 +365,7 @@ export const groupBySelections = (
       groupByLabel = 'category';
       break;
     default:
-      throw new Error('Error loading data into the spreadsheet.');
+      throw new Error(t('Error loading data into the spreadsheet.'));
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏exception messages should not be translated IMO
```suggestion
      throw new Error('Error loading data into the spreadsheet.');
```

Reply:
I would've found it odd displaying them in other language but after googling it, it makes sense 😅
Thanks! I'll change it